### PR TITLE
docs: Add udev rules documentation to `README.md`

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,8 +37,17 @@ The following Logitech Litra devices, __connected via USB__, are supported:
 ### All other platforms (via binary)
 
 1. Download the [latest release](https://github.com/timrogers/litra-rs/releases/latest) for your platform. macOS, Linux and Windows devices are supported.
-1. Add the binary to `$PATH`, so you can execute it from your shell. For the best experience, call it `litra` on macOS and Linux, and `litra.exe` on Windows.
-1. Run `litra --help` to check that everything is working and see the available commands.
+2. Add the binary to `$PATH`, so you can execute it from your shell. For the best experience, call it `litra` on macOS and Linux, and `litra.exe` on Windows.
+3. Run `litra --help` to check that everything is working and see the available commands.
+
+## Permission on Linux
```
Comment (timrogers) [inline README.md:43]:
```suggestion
## Configuring `udev` permissions on Linux
```

File: README.md [hunk 2]
```diff
@@ -37,8 +37,17 @@ The following Logitech Litra devices, __connected via USB__, are supported:
...
+
+On Linux, you can configure the permission for the Litra devices using [udev](https://www.man7.org/linux/man-pages/man7/udev.7.html).
+To allow allow all users that are part of the `video` group to access the Litra devices, copy the [`99-litra.rules`](99-litra.rules) file into `/etc/udev/rules.d`.
```
Comment (timrogers) [inline README.md:46]:
```suggestion
To allow all users that are part of the `video` group to access the Litra devices, copy the [`99-litra.rules`](99-litra.rules) file into `/etc/udev/rules.d`.
```

File: README.md [hunk 3]
```diff
@@ -37,8 +37,17 @@ The following Logitech Litra devices, __connected via USB__, are supported:
...
+Afterwards, reboot your computer or run the following commands as `root` user:
```
Comment (timrogers) [inline README.md:47]:
```suggestion
Next, reboot your computer or run the following commands as `root`:
```

File: README.md (hunk 2 above)
Comment (timrogers) [inline README.md:46]: If I don't do this, what is the default state? Is it that only `root` can manipulate the Litra devices?

**Review by timrogers — COMMENTED** [review]
Thanks for taking the time to put this together! I'm going to make a couple of copy tweaks, and I have a question.

File: README.md (hunk 2 above)
Comment (Holzhaus) [reply README.md]: Yes (unless your distro ships udev rules that set hidraw devices to mode `0666` or something like that). For me, `litra` crashed with "Permission Error" when trying to run it as non-root user.

File: README.md [hunk 4]
```diff
@@ -37,8 +37,17 @@ The following Logitech Litra devices, __connected via USB__, are supported:
...
 1. Download the [latest release](https://github.com/timrogers/litra-rs/releases/latest) for your platform. macOS, Linux and Windows devices are supported.
-1. Add the binary to `$PATH`, so you can execute it from your shell. For the best experience, call it `litra` on macOS and Linux, and `litra.exe` on Windows.
-1. Run `litra --help` to check that everything is working and see the available commands.
+2. Add the binary to `$PATH`, so you can execute it from your shell. For the best experience, call it `litra` on macOS and Linux, and `litra.exe` on Windows.
+3. Run `litra --help` to check that everything is working and see the available commands.
+
+## Configuring `udev` permissions on Linux
+
+On Linux, you can configure the permission for the Litra devices using [udev](https://www.man7.org/linux/man-pages/man7/udev.7.html).
+To allow all users that are part of the `video` group to access the Litra devices, copy the [`99-litra.rules`](99-litra.rules) file into `/etc/udev/rules.d`.
```
Comment (timrogers) [inline README.md:46]:
```suggestion
On most Linux operating systems, you will need to manually configure permissions using [`udev`](https://www.man7.org/linux/man-pages/man7/udev.7.html) to allow non-`root` users to access and manage Litra device's.

To allow all users that are part of the `video` group to access the Litra devices, copy the [`99-litra.rules`](99-litra.rules) file into `/etc/udev/rules.d`.
```

**Pull request merged**